airbyte-ci/connectors/CONNECTOR_CHECKLIST.yaml

```
@@ -0,0 +1,12 @@
+paths:
+  "airbyte-integrations/connectors/**":
+    - PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
+    - Connector version has been incremented in the Dockerfile and metadata.yaml according to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines
+    - "[Breaking changes are considered](https://docs.airbyte.com/contributing-to-airbyte/#breaking-changes-to-connectors). If a **Breaking Change** is being introduced, ensure an Airbyte engineer has created a Breaking Change Plan and you've followed all steps in the [Breaking Changes Checklist](https://docs.airbyte.com/contributing-to-airbyte/#checklist-for-contributors)"
+    - Secrets in the connector's spec are annotated with `airbyte_secret`
+    - Ensure all documentation files are up to date. (README.md, bootstrap.md, docs.md, etc...)
```

```suggestion
    - All documentation files are up to date. (README.md, bootstrap.md, docs.md, etc...)
```
nit: phrasing consistency